Tests/Functional/xctest_checker/xctest_checker/compare.py

```diff
@@ -28,8 +28,10 @@ def _expected_lines_and_line_numbers(path, check_prefix):
     with open(path) as f:
         for line_number, line in enumerate(f):
             if line.startswith(check_prefix):
-                yield line[len(check_prefix):], line_number+1
+                yield line[len(check_prefix):].strip(), line_number+1
 
+def _add_whitespace_leniency(original_regex):
+    return "^ *" + original_regex + " *$"
```

Comment:
Will the spaces here only match spaces, or will they match tabs as well? Perhaps use `\s`?